protocol/dubbo/impl/codec.go

```diff
@@ -163,15 +163,12 @@ func (c *ProtocolCodec) Decode(p *DubboPackage) error {
 	if p.IsResponseWithException() {
 		logger.Infof("response with exception: %+v", p.Header)
 		decoder := hessian.NewDecoder(body)
+		p.Body = &ResponsePayload{}
```

Comment:
why？

Reply:
here, p.Body also be nil, so will return perrors.Errorf("java exception:%s", exception.(string)), this will cause the client timeout always.